src/background.ts

```diff
@@ -111,40 +113,62 @@ async function stateChangeCallback(newState: String) {
       webScience.pageManager.onPageVisitStart.removeListener(pageVisitStartListener);
       webScience.pageManager.onPageVisitStop.removeListener(pageVisitStopListener);
 
-      await browser.storage.local.set({ "state": runStates.PAUSED });
+      await browser.storage.local.set({ "state": RunStates.Paused });
 
       break;
     }
   }
 }
 
-const schemaNamespace = "rally-markup-fb-pixel-hunt";
 // Initialize the Rally SDK.
-const rally = new Rally();
-rally.initialize(schemaNamespace, publicKey, enableDevMode, stateChangeCallback).then(() => {
-  // The Rally Core Add-on expects the extension to automatically start, unlike the new Web Platform SDK.
-  stateChangeCallback("resume");
-
-  // When in developer mode, open the options page with the playtest controls.
-  if (enableDevMode) {
-    browser.runtime.onMessage.addListener((m, s) => {
-      if (!("type" in m && m.type.startsWith("rally-sdk"))) {
-        // Only listen for messages from the rally-sdk.
-        return;
-      }
-      if (m.data.state === "resume") {
-        stateChangeCallback("resume")
-      } else if (m.data.state === "pause") {
-        stateChangeCallback("pause")
-      } else {
-        throw new Error(`Unknown state: ${m.data.state}`);
-      }
-    });
+const studyId = "facebookPixelHunt";
+const rallySite = "http://localhost:3000";
 
-    browser.storage.local.set({ "state": runStates.PAUSED }).then(() =>
-      browser.storage.local.set({ "initialized": true }).then(() =>
-        browser.runtime.openOptionsPage()
-      )
-    );
-  }
-});
\ No newline at end of file
+/**
+ * Firebase config for staging.
+ */
+const firebaseConfig = {
+  "apiKey": "AIzaSyAj3z6_cRdiBzwTuVzey6sJm0hVDVBSrDg",
+  "authDomain": "moz-fx-data-rall-nonprod-ac2a.firebaseapp.com",
+  "projectId": "moz-fx-data-rall-nonprod-ac2a",
+  "storageBucket": "moz-fx-data-rall-nonprod-ac2a.appspot.com",
+  "messagingSenderId": "451372671583",
+  "appId": "1:451372671583:web:eeb61e7d7c8ec898f5b1ea",
+  "functionsHost": "https://us-central1-moz-fx-data-rall-nonprod-ac2a.cloudfunctions.net"
+};
+
+const rally = new Rally({
+  studyId,
+  rallySite,
+  firebaseConfig,
+  enableEmulatorMode: enableDevMode,
+  enableDevMode,
+  stateChangeCallback,
+});
+
+// The Rally Core Add-on expects the extension to automatically start, unlike the new Web Platform SDK.
+stateChangeCallback(RunStates.Running);
```

Comment:
@rhelmer I am not sure if this should be omitted in new SDK. Please advise. If this is removed then we need to change the integration tests to start in paused state.

Reply:
Yes, I think the extension needs to start paused, because we don't want data collection to begin until it's able to log in and confirm that the user has given consent to the study.

Reply:
Thanks for confirming. I had a suspicion this may be the case. I will remove this part and change integration tests accordingly.